src/Cooked/Ltl/Combinators.hs

```diff
@@ -0,0 +1,31 @@
+-- | This module provides helpers for writing common LTL expressions.
+module Cooked.Ltl.Combinators
+  ( anyOf,
+    allOf,
+    anyOf',
+    allOf',
+  )
+where
```

Comment:
I like this module. I think a few other combinators should be added in it:
* `ltlDelay`, to be moved from module `Cooked.Ltl` . It could be renamed just `delay` since other combinators are not prefixed by ltl. Alternatively, all combinators could be. 
* eventually (which would correspond to applying `LtlUntil LtlTruth` on a formula)
* always (which would correspond to applying `LtlRelease LtlFalsity` on a formula)